bitnami/spark/templates/statefulset-master.yaml

```
@@ -226,6 +231,8 @@ spec:
               value: {{ .Values.security.ssl.needClientAuth | quote }}
             - name: SPARK_SSL_PROTOCOL
               value: {{ .Values.security.ssl.protocol | quote }}
+            - name: SPARK_WEBUI_SSL_PORT
+              value: {{ .Values.master.webPortHttps }}
```

Missing quotes will cause the Statefulset to fail
```suggestion
              value: {{ .Values.master.webPortHttps | quote }}
```